#!/usr/env/python

PACKAGE = "fg_project1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

euler = gen.const("euler", int_t, 0, "euler integration")
rk = gen.const("rk", int_t, 1, "rungekutta integration")

int_enum = gen.enum([euler, rk], "integration method selection")

gen.add("odom_int", int_t, 1, "integration method for odometry", 0, 0, 1, edit_method=int_enum)

exit(gen.generate("fg_project1", "odom", "intparams"))